.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,9 @@
 ### Description of changes
 
-Please describe your pull request
+Please describe your pull request.
 
 ### Checklist
-```[tasklist]
-- [ ] Written tests for the changes
-```
 
-### Optional tasks
-```[tasklist]
+- [ ] Written tests for the changes
 - [ ] Update k6 documentation (if relevant)
```

Comment:
Maybe make it a little more explicit by asking for the PR link :) Not sure what the best way though is.

```suggestion
- [ ] Update k6 documentation (if relevant) -- PR link: ?
```

Reply:
That look good 🙃